packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -136,6 +142,7 @@ const SageMaker = {
   'eu-west-3': 'eu-west-3',
   'eu-central-1': 'eu-central-1',
   'eu-north-1': 'eu-north-1',
+  'eu-south-1': 'eu-south-1',
```

Comment:
SageMaker is available in eu-south-1.

https://docs.aws.amazon.com/general/latest/gr/sagemaker.html